templates/zerver/for/open-source.md

```diff
@@ -1,104 +1,92 @@
-  your community.
-
-* **Walls optional:** Take down the barrier to participation created
-  by making people interested in your community ask for an invitation
-  before they can enter — use [one
-  checkbox](/help/allow-anyone-to-join-without-an-invitation) to open
-  your Zulip organization for anyone to join.
-
-* **Badges:** Want to link to your Zulip organization with a nice
-  badge in your README?  [Done.](/help/join-zulip-chat-badge)
-
-<!--- This needs better documentation. -->
-* **GitHub auth?** [We've got that.][github-auth]
-
+The Zulip core developers have decades of combined experience building,
+running, and maintaining free and open source software. We use Zulip to keep
+contributors engaged, efficiently make decisions, and fashion the day-to-day
+experience of being a part of our project. There is no other chat product
+that comes even close to Zulip in facilitating contributor engagement,
+making efficient use of maintainers' time, and upholding the values of the
+FOSS community.
+
+If you haven't read [why Zulip](/why-zulip), read that first. We've also
+collected a list of features we think will be of particular interest for
+open source communities using Zulip.
+
+**Free hosting at zulipchat.com**
+
+No catch; the hosting is supported by (and is identical to) zulipchat.com's
+commercial offerings. This offer extends to any community involved in
+supporting free and open source software: foundations, maintainer groups,
+hackathons, meetups, and others. If you’re not sure whether your
+organization qualifies, send us an email at support@zulipchat.com.
+
+**Scale to thousands of users**
+
+For those running Zulip at home, we’ve done a lot of work to ensure that
+drive-by members of the community don’t consume too many disk or CPU
+resources. A Zulip server with 1000 active members and 10000 inactive
+members takes about as many resources as a Zulip server with 1000 active
+members.
+
+**Join without an invitation**
+
+Allow anyone to
+[join without an invitation](/help/allow-anyone-to-join-without-an-invitation).
+You can also link to your Zulip with a [badge](/help/join-zulip-chat-badge)
+in your readme document.
+
+**Authenticate with GitHub**
+
+Allow (or require) users to
+[authenticate with their GitHub account][github-auth], instead of with their
+email address.
 [github-auth]: https://github.com/zulip/zulip/blob/7e9926233/zproject/prod_settings_template.py#L112
 
-* **Scale:** Zulip scales well to large communities — it's loved by
-  communities as large as 1000 users and beyond.
+**Quality data export**
+
+Our high quality export and import tools ensure you can always move from
+[zulipchat.com](https://zulipchat.com) hosting to your own servers.
+
+**Syntax highlighting**
+
+[Full Markdown support](/help/format-your-message-using-markdown), including
+syntax highlighting, makes it easy to discuss code, paste an error message,
+or make a complicated argument. Full LaTeX support as well.
```

Comment:
I worry a bit about people thinking "argument" means "heated discussion", especially if their English isn't strong and/or they just haven't spent as much time with academics as you and me. :-P Maybe s/argument/point/.